components/dropdown/src/auro-dropdownBib.js

```diff
@@ -40,6 +50,21 @@ export class AuroDropdownBib extends LitElement {
       rounded: {
         type: Boolean,
         reflect: true
+      },
+      mobileFullscreenBreakpoint: {
+        type: String,
+        converter: (value) => {
+          // verify the defined breakpoint is valid and exit out if not
+          const validatedValue = DESIGN_TOKEN_BREAKPOINT_OPTIONS.includes(value) ? value : undefined;
+          if (!validatedValue) {
+            return undefined;
+          }
+
+          // get the pixel value for the defined breakpoint
+          const docStyle = getComputedStyle(document.documentElement);
+          return docStyle.getPropertyValue(DESIGN_TOKEN_BREAKPOINT_PREFIX + value);
+        },
```

Comment:
I wonder if this should be moved to a function to keep the constructor clean. Thoughts?

```suggestion
        converter: getBreakpointValue()
```